ArgusCore/src/test/java/com/salesforce/dva/argus/service/metric/transform/HoltWintersTransformTest.java

```diff
@@ -31,15 +31,31 @@
      
 package com.salesforce.dva.argus.service.metric.transform;
 
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
 import org.junit.Test;
 import java.util.HashMap;
+import java.util.Locale;
 import java.util.Map;
 
 import static org.junit.Assert.*;
 
 public class HoltWintersTransformTest {
 
-    @Test
+	private static Locale locale;
```

Comment:
Revert this, it has been handled in a separate PR that has already been merged: https://github.com/salesforce/Argus/pull/337